src/DataDefinitionsBundle/Filter/FilterInterface.php

```diff
@@ -21,5 +21,5 @@
 
 interface FilterInterface
 {
-    public function filter(DataDefinitionInterface $definition, array $data, Concrete $object): bool;
```

Comment:
TBH I'm not in love with these interfaces having 6 parameters. Can we maybe change the `DataDefinitionInterface` here?

Reply:
so basically introducing a sort of `ImportContextInterface`? Where the state of the current run is stored?

Reply:
For example, or we replace the `$data` param with something more substantial which *provides* data. Otherwise we'll keep adding params in these interfaces forever.

Interpreter interface has the same issue, it has 6 params IIRC.

Reply:
$data is the current Row of the import as actual Data.

Reply:
IMO, we should rethink these interfaces before 4.0 final. I know it's painful to do that instead of just appending another argument, but it will only get more painful in the future. :)

Reply:
I'll do another PR for that ok?

Reply:
and solve it all in one PR